pkg/client/options.go

```diff
@@ -948,3 +973,57 @@ func (o *DeleteAllOfOptions) ApplyToDeleteAllOf(do *DeleteAllOfOptions) {
 }
 
 // }}}
+
+// ApplyOptions are the options for an apply request.
+type ApplyOptions struct {
+	// When present, indicates that modifications should not be
+	// persisted. An invalid or unrecognized dryRun directive will
+	// result in an error response and no further processing of the
+	// request. Valid values are:
+	// - All: all dry run stages will be processed
+	// +optional
+	// +listType=atomic
+	DryRun []string `json:"dryRun,omitempty" protobuf:"bytes,1,rep,name=dryRun"`
+
+	// Force is going to "force" Apply requests. It means user will
+	// re-acquire conflicting fields owned by other people.
```

Comment:
Looks like // +optional and omitempty is missing

cf.
```go
	// +optional
	Force *bool `json:"force,omitempty" protobuf:"varint,2,opt,name=force"`
```
(metav1.PatchOptions)

Reply:
Ah that's fine. I thought we somehow need the JSON tags because we marshal it